onnxruntime/test/onnx/TestCase.h

```diff
@@ -52,6 +53,7 @@ class TestModelInfo {
   virtual int GetOutputCount() const = 0;
   virtual const std::string& GetInputName(size_t i) const = 0;
   virtual const std::string& GetOutputName(size_t i) const = 0;
+  virtual std::string GetModelVersion() const {return "";}
```

Comment:
>const {return "" [](start = 40, length = 16)

Make it pure

Reply:
TestModelInfo has multiple subclasses, providing a default implementation here avoid needless changes.